src/url.js

```diff
@@ -207,6 +207,20 @@ export function isProxyOrigin(url) {
 }
 
 /**
+ * Removes parameters that start with the specified pattern and returns the new
+ * search string.
+ * @param {string} urlSearch
+ * @param {string} param
+ * @return {string}
+ */
+function removeMatchingParams(urlSearch, pattern) {
```

Comment:
Not sure if I need to keep pattern as a parameter then. Can simply rename the function to `removeAmpJsParams()`. does that work? then I'll move regex to a const and if we need it to be extensible, we can redo this.